package.json

```
@@ -22,7 +22,7 @@
     "reinstall": "yarn run clean && yarn install"
   },
   "engines": {
-    "node": ">=22.0.0"
+    "node": ">=20.0.0"
```

```suggestion
    "node": ">=22.0.0"
```

To unify with other CKEditor 5 repositories.